java/src/jmri/jmrix/can/cbus/CbusConsistManager.java

```diff
@@ -0,0 +1,87 @@
+package jmri.jmrix.can.cbus;
+
+import java.util.Vector;
+
+import jmri.implementation.*;
+import jmri.AddressedProgrammerManager;
+import jmri.CommandStation;
+import jmri.ConsistManager.EnableListener;
+
+/**
+ * The CBUS Consist Manager which uses the NmraConsist class for
+ * the consists it builds. This implementation just tracks the
+ * consist via a table of {@link jmri.implementation.DccConsist} objects
+ * that handle the actual operations.
+ *
+ * @author Andrew Crosland Copyright (C) 2022
+ */
+public class CbusConsistManager extends NmraConsistManager implements EnableListener {
+
+    public CbusConsistManager(CommandStation cs) {
+        super(cs);
+    }
+
+    /**
+     * Can this consist manager be disabled?
+     * 
+     * @return true if the manager can be disabled, false otherwise
+     */
+    @Override
+    public boolean canBeDisabled() {
+        return true;
+    }
+
+    protected static final Vector<EnableListener> enableListeners = new Vector<EnableListener>();
```

Comment:
I suggest using `List` and `ArrayList` instead of `Vector`. You can synchronize the methods that's using `enableListeners` instead.

The `Vector` class is deprecated since it's synchronized.